.github/workflows/integration-tests-v2.yml

```diff
@@ -118,4 +103,4 @@ jobs:
           FIREBOLT_CLIENT_ID: ${{ secrets.FIREBOLT_CLIENT_ID_STG_NEW_IDN  }}
           FIREBOLT_CLIENT_SECRET: ${{ secrets.FIREBOLT_CLIENT_SECRET_STG_NEW_IDN  }}
         run: |
-          dotnet test --filter '(Category=Integration|Category=IntegrationSetup)&(Category=${{ inputs.category }}|Category=general)&Name!=ExecuteLongTest' -l "console;verbosity=normal"
+          dotnet test --filter '(Category=Integration|Category=IntegrationSetup)&(Category=engine-v2|Category=general)&Name!=ExecuteLongTest' -l "console;verbosity=normal"
```

Comment:
should we run both `v2` and `engine-v2` here?

Reply:
I think tests that are compatible with engines v2 are already marked as `engine-v2` and those that don't are marked only `v2`. Some tests have both categories so the line in question will run them.